### ISSUE TITLE
Filter applied in get() returns 400065: Bad Request

### ISSUE THREAD

**Issue by ludwigrubio** [tableau/server-client-python#753]
I'm trying to filter by name a user, group or project, I'm getting following error message.

	400065: Bad Request
		The filter query contains a key which is not recognized for this api version.

**Environment:**
**server-client-python  version:** 0.14.0
**server version**: 3.2 (Not working in any version from 2.0 to 3.2)

**Code used:**
```
tableau_auth = TSC.TableauAuth(user, password, site_name)
server = TSC.Server('https://mysite.com')
server.use_server_version() #3.2

with server.auth.sign_in(tableau_auth):
        req_option = TSC.RequestOptions()
        req_option.filter.add(TSC.Filter(TSC.RequestOptions.Field.Name,
                                     TSC.RequestOptions.Operator.Equals,
                                     'THE_USER_NAME'))
       all_entities, pagination_item = server.users.get(req_option)
```

**Comment by bcantoni** [comment]
@ludwigrubio what's the version of Tableau Server that you are connecting to? I've done a few tests on 2020.4 and getting users with the name filter seems to work fine, but it might be version-specific.

**Comment by ludwigrubio** [comment]
@bcantoni Tableau Server Version: 2018.3.4 64-bit Windows

**Comment by shinchris** [comment]
Looks like the same issue as #732

**Comment by shinchris** [comment]
This should be fixed in v0.14.1.
